scss/components/_buttons.scss

```diff
@@ -0,0 +1,15 @@
+.btn {
+    background-color: $primary-color;
+    color: #fff;
+    padding: 1em 3em;
+    text-transform: uppercase;
+    font-size: 0.8rem;
```

Comment:
I think it would be great to have variables for all the colors and font sizes used in your project. It's the best way to write sass. That way, in future, if this has to go open-source, others can completely customise the app by just updating scss variables.

Reply:
> @Monday6984 for the round text, some others exported it as an image. I think you can export it as png or svg. Much easier than coding it.

Ah! I never thought of exporting it. Thank you.

As regards the font-sizes to be declared as variables, you know I would still adjust the font-sizes when working on the responsiveness. That is why I didn't add them as variables but let me see how it goes as I progress.